db/migrations/20210806122814_add_number_of_items_to_playlist_migration.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Add a new column (maxNumberOfItems) to Playlist table
+ *
+ * @phpcs:disable PSR1.Classes.ClassDeclaration.MissingNamespace
+ */
+
+use Phinx\Migration\AbstractMigration;
+
+class AddNumberOfItemsToPlaylistMigration extends AbstractMigration
+{
+    /** @inheritDoc */
+    public function change()
+    {
+        $this->table('playlist')
+            ->addColumn('maxNumberOfItems', 'integer', ['default' => null, 'null' => true, 'after' => 'filterMediaTags'])
+            ->save();
+
+        $this->table('setting')->insert([
+            [
+                'setting' => 'DEFAULT_DYNAMIC_PLAYLIST_MAXNUMBER',
+                'value' => 30,
+                'userSee' => 1,
+                'userChange' => 1
+            ],
+            [
+                'setting' => 'DEFAULT_DYNAMIC_PLAYLIST_MAXNUMBER_LIMIT',
+                'value' => 100,
+                'userSee' => 1,
+                'userChange' => 1
+            ]
+        ])->save();
+
+        $this->execute('UPDATE `playlist` SET maxNumberOfItems = 30 WHERE isDynamic = 1');
```

Comment:
Do we want to try and count them do you think? So that we don't break use cases on upgrade?

Untested SQL incoming...

```
SELECT COUNT(*) AS cnt, widget.playlistId
  FROM `widget` 
 WHERE playlistId IN (SELECT playlistId FROM `playlist` WHERE isDynamic = 1)
GROUP BY widget.playlistId
ORDER BY COUNT(*) DESC
```

Could we then use `min($record[0]['cnt'], 30)` and `min($record[0]['cnt'], 100)` respectively?